File: Classical Date System/PHP/penanggalan.php
<?php
    $thn = date("Y");
    $bln = date("n");
    $bul = date("F");
    $tgl = date("j");
    $hari = date("l");
    $harke = date("z");
    $jam = date("H");
    $mnt = date("i");
    $dtk = date("s");

    echo "Tahun                   : ".$thn."\n";
    echo "Bulan                   : ".$bln." (".$bul.")"."\n";
    echo "Tanggal                 : ".$tgl."\n";
    echo "Hari                    : ".$hari."\n";
    echo "Hari ke (dalam Setahun) : ".$harke."\n";
    echo "Jam                     : ".$jam."\n";
    echo "Menit                   : ".$mnt."\n";
    echo "Detik                   : ".$dtk."\n";
    
    echo "\n".date("l, j F Y")."\n";
    echo "Day-".$harke."\n";
    echo date("H:i:s");
?>

File: Classical Date System/Dart/penanggalan.dart
import 'dart:io';
import 'package:intl/intl.dart';

void main() {
  DateTime datetime = DateTime.now();

  String thn = datetime.year.toString();
  String bln = datetime.month.toString();
  String bul = DateFormat.MMMM().format(datetime);
  String tgl = datetime.day.toString();
  String har = datetime.weekday.toString();
  String hari = DateFormat.EEEE().format(datetime);
  int hariKe = int.parse(DateFormat("D").format(datetime));
  String jam = datetime.hour.toString().padLeft(2, '0');
  String mnt = datetime.minute.toString().padLeft(2, '0');
  String dtk = datetime.second.toString().padLeft(2, '0');

  print("Tahun                   : $thn");
  print("Bulan                   : $bln ($bul)");
  print("Tanggal                 : $tgl");
  print("Hari                    : $har ($hari)");
  print("Hari ke (dalam Setahun) : $hariKe");
  print("Jam                     : $jam");
  print("Menit                   : $mnt");
  print("Detik                   : $dtk\n");

  // Format lengkap
  print(DateFormat("EEEE, d MMMM yyyy").format(datetime));
  print("Day-$hariKe");
  print("$jam:$mnt:$dtk");
}
